CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # splat Release Notes
 
+* splat now checks if symbol names can be valid filepaths and produce an error if not.
+  * This is checked because functions are written to their own files and the symbol name is used as the filepath.
+  * There are two checks in place:
+    * The resulting filepath should not exceed 255 bytes, since most OSes impose that restriction.
```

Comment:
is it just the name or the full path? I thought it was the name only

Reply:
definitely filename not filepath